src/docGeneratorMain.ts

```diff
@@ -210,9 +210,16 @@ async function main() {
   await docGenerator.downloadDocs(
     (pattern) =>
       `${githubBaseUrl}/sindresorhus/eslint-plugin-unicorn/main/docs/rules/${pattern}.md`,
-    "unicorn",
+    "eslint-plugin-unicorn",
```

Comment:
This change here is wrong. This is not the name of the package but the name of the plugin, which is stated in the plugin's documentation: 
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/readme.md#usage
```json
		"plugins": [
			"unicorn"
		]

```